docs/libraries/nodejs/examples.md

```diff
@@ -1,101 +1,149 @@
 # Examples
 
-There are several examples to show the usage of the library.
-
-> All examples can be found in [/bindings/nodejs/examples](https://github.com/iotaledger/wallet.rs/bindings/nodejs/examples/)
-
-## Setup
-First, setup your environment as follows:
+There are several examples to show the usage of the library. All examples below can be also found in [/bindings/nodejs/examples](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/nodejs/examples)
 
+All examples below expect your custom password in the `.env` file:
```

Comment:
we should probably warn that this is not a secure way of doing it, and maybe they should put their password on a `.bashrc` file or load the password from a source other than the environment variables.

Reply:
Great point. Added a notice and reference to security and backup best practices from Chrysalis Docs
https://github.com/iotaledger/wallet.rs/pull/437/commits/c502d8c7b155310f6d2f44aa41ec0320163a976e